libbeat/publisher/scheduling/ratelimit/config.go

```diff
@@ -0,0 +1,22 @@
+package ratelimit
+
+import "errors"
+
+type config struct {
+	EventsPerSecond uint `config:"events_per_second" validate:"min=1"`
```

Comment:
Should we make the time unit configurable instead of hardcoding it "per second"? It would mean to split up this config into 2 parts, one for the duration and one for the number of events.